pkg/ipamd/ipamd.go

```diff
@@ -770,11 +770,16 @@ func (c *IPAMContext) tryUnassignCidrsFromAll() {
 	if over > 0 {
 		eniInfos := c.dataStore.GetENIInfos()
 		for eniID := range eniInfos.ENIs {
+			// if the deallocated target is achieved, we can exit
+			if over <= 0 {
+				break
+			}
```

Comment:
nit: this check would be cleaner at the end of the loop, i.e. after we decrease `over`

Reply:
sure, done